EndpointCategory/helpers/instantiateEndpoint.js

```diff
@@ -87,84 +87,114 @@ module.exports = (config) => {
-          }
-        } else if (body.message) {
+          } else if (body.message) {
           // Error message not in the error map so check body
-          if (typeof body.message === 'string') {
-            zoomErrorMessage = body.message;
+            if (typeof body.message === 'string') {
+              zoomErrorMessage = body.message;
+            }
           }
-        }
-        // Note: if none of the conditionals hit,
-        // zoomErrorMessage defaults to generic error message
+          // Note: if none of the conditionals hit,
+          // zoomErrorMessage defaults to generic error message
 
-        const errorMessage = `We couldn't ${action} because an error occurred: ${zoomErrorMessage}`;
-        const errorCode = `ZOOM${status}${body.code ? `-${body.code}` : ''}`;
+          const errorMessage = `We couldn't ${action} because an error occurred: ${zoomErrorMessage}`;
+          const errorCode = `ZOOM${status}${body.code ? `-${body.code}` : ''}`;
 
-        throw new ZACCLError({
-          message: errorMessage,
-          code: errorCode,
-        });
-      }
+          throw new ZACCLError({
+            message: errorMessage,
+            code: errorCode,
+          });
+        }
 
-      // Run the post-processor, throwing any errors it produces
-      // and convert non-ZACCLError errors into ZACCLErrors with better text
-      let modifiedResponse = response;
-      if (postProcessor) {
-        try {
-          modifiedResponse = postProcessor(response);
-        } catch (err) {
+        // Run the post-processor, throwing any errors it produces
+        // and convert non-ZACCLError errors into ZACCLErrors with better text
+        let modifiedResponse = response;
+        if (postProcessor) {
+          try {
+            modifiedResponse = postProcessor(response);
+          } catch (err) {
           // Turn into ZACCLError if not already
-          let newError = err;
-          if (!err.isZACCLError) {
-            newError = new ZACCLError(err);
-            newError.message = 'An error occurred while post-processing the response from Zoom.';
-            newError.code = ERROR_CODES.POST_PROCESSING_ERROR;
+            let newError = err;
+            if (!err.isZACCLError) {
+              newError = new ZACCLError(err);
+              newError.message = 'An error occurred while post-processing the response from Zoom.';
+              newError.code = ERROR_CODES.POST_PROCESSING_ERROR;
+            }
+            throw newError;
           }
-          throw newError;
         }
-      }
-      // Return the body of the zoom response
-      return modifiedResponse;
+
+        // Response is valid. Add response to pages
+        pages.push(modifiedResponse);
+
+        // Check for next page
+        if (modifiedResponse.body.next_page_token) {
+          // Add next page token to the params
+          endpointOptions.params
+            .next_page_token = modifiedResponse.body.next_page_token;
```

Comment:
Is this newline intentional?

Reply:
My guess is this is due to the 80char limit. If it is, here's the preferred format:

```
endpointOptions.params.next_page_token = (
  modifiedResponse.body.next_page_token
);
```